Source/WindowsShellExtension/dllmain.cpp

```diff
@@ -0,0 +1,293 @@
+// dllmain.cpp : Defines the entry point for the DLL application.
+
+// IExplorerCommand File Explorer shell extension for context menu entry
+
+#include "pch.h"
+
+#ifdef MEDIAINFO_QT
+    #define SHELLEXT_GUID "dea3006d-451d-4f0f-a2b2-c3250cb255b4"
+#else
+    #define SHELLEXT_GUID "20669675-b281-4c4f-94fb-cb6fd3995545"
```

Comment:
Not possible to have 1 GUID for both?

Reply:
Going to have issues if both are installed at the same time. There will be two entries but both launch the same one.